### PR TITLE
[17.0][IMP]purchase_order_type: Automatically get partner purchase type on purchase creation.

### PR DESCRIPTION
**Pull request by RogerSans**
Automatically get partner purchase type on purchase creation.
T-6445

@HaraldPanten 
@ValentinVinagre 
@luis-ron

### CODE REVIEW

File: purchase_order_type/tests/test_purchase_order_type.py [hunk 1]
```diff
@@ -88,3 +88,21 @@ def test_purchase_order_type_company_error(self):
         self.assertEqual(order.company_id, self.type1.company_id)
         with self.assertRaises(ValidationError):
             order.write({"company_id": self.company2.id})
+
+    def test_order_type_from_partner(self):
```
Comment (ValentinVinagre) [inline purchase_order_type/tests/test_purchase_order_type.py:92]: do more tests please 😄

Comment (RogerSans) [reply purchase_order_type/tests/test_purchase_order_type.py:92]: Can you review pls? THX

**Comment by ValentinVinagre** [comment]
@luis-ron could u review? 😄

**Review by luis-ron — APPROVED** [review]
Functional Review: LGTM 👍🏻

**Review by ValentinVinagre — APPROVED** [review]
LGTM 👍🏻

**Comment by ValentinVinagre** [comment]
@RogerSans add the versión of odoo in the pr title please.

**Comment by RogerSans** [comment]
> @RogerSans add the versión of odoo in the pr title please.

Done

**Review by HaraldPanten — APPROVED** [review]

**Pull request merged**